contributing_to_docs/doc_guidelines.adoc

```diff
@@ -484,6 +484,32 @@ $ lorem.sh
 ----
 ....
 
+* Use only one command per code block. For example, do not include more than one command in the same code block:
++
+Run the following commands to create templates you can modify:
++
+----
+$ oc adm create-login-template > login.html
+$ oc adm create-provider-selection-template > providers.html
+$ oc adm create-error-template > errors.html
+----
```

Comment:
This is confusing to me. It is saying to only use one, then it's saying for example, not more than one. Then the example is something we shouldn't be doing. 

I suggest either:
A) Changing the guideline to something like, "Do not use more than one command per code block. For example, the following should be split up into three separate code blocks."
B) Change the example to show a single-command code block, following the guideline as it's laid out.